src/vs/vscode.d.ts

```diff
@@ -2419,7 +2419,7 @@ declare module 'vscode' {
 	/**
 	 * Information about where a symbol is defined.
 	 *
-	 * Provides additional metadata over normal [location](#Location) definitions, including the range of
+	 * Provides additional metadata over normal {@link Location location} definitions, including the range of
```

Comment:
@gregvanl Once we support `@link` in the editor, I'd like to update `vscode.d.ts` to use `@link` instead of our custom link scheme. This means we'd also need to update the website to understand these links when generating documentation

Reply:
@jrieken I think it makes sense to adopt `@link` as much as possible `vscode.d.ts`. Here's my proposal for how to do this:

1. **April**: Merge support for `@link`  
2. **May**: Pick up TS 4.3. Start updating `vscode.d.ts` to use `@link` tags.
3. **May**: Ensure the website's doc generator can parse `@link` tags.
4. **May Release**: Users targeting the latest VS Code version get the `vscode.d.ts` that uses `@link`. However users targeting older versions of VS Code will get the `d.ts` that uses our custom link scheme
5. **Post May**: All new llinks added to `vscode.d.ts` should use `@link`
6. **Future?**: Remove custom link support? This would break `#link` in old `vscode.d.ts` versions